web/tests/typeahead_helper.test.cjs

```diff
@@ -1140,11 +1151,51 @@ test("sort_group_setting_options", ({override_rewire}) => {
         admins_group.name,
         a_user.full_name,
         bob_system_group.name,
+        members_group.name,
+        bob_group.name,
+        second_bob_group.name,
+        b_user_2.full_name,
+        b_user_1.full_name,
+        b_user_3.full_name,
+    ]);
+
+    assert.deepEqual(get_group_setting_typeahead_result("me", second_bob_group), [
+        members_group.name,
+        bob_system_group.name,
+        admins_group.name,
+        bob_group.name,
+        second_bob_group.name,
+        b_user_2.full_name,
+        a_user.full_name,
+        b_user_1.full_name,
+        b_user_3.full_name,
+        zman.full_name,
+    ]);
+
+    assert.deepEqual(get_group_setting_typeahead_result("ever", second_bob_group), [
+        members_group.name,
+        bob_system_group.name,
+        admins_group.name,
+        bob_group.name,
+        second_bob_group.name,
+        b_user_2.full_name,
+        a_user.full_name,
+        b_user_1.full_name,
+        b_user_3.full_name,
+        zman.full_name,
+    ]);
+
+    assert.deepEqual(get_group_setting_typeahead_result("translated: members", second_bob_group), [
```

Comment:
Added this to complete coverage for the exact match case and `translated:` prefix is required for that.